Custom/Internal/AzureAsyncCollectionResult.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System.ClientModel;
```

Comment:
These two files look like they accidentally landed in the root rather the `/sdk/openai` area.

Reply:
*sigh* Applying patch files can do some strange things sometimes. Thanks for catching this!